orchagent/vnetorch.h

```diff
@@ -0,0 +1,197 @@
+#ifndef __VNETORCH_H
+#define __VNETORCH_H
+
+#include <vector>
+#include <set>
+#include <unordered_map>
+#include <algorithm>
+
+#include "request_parser.h"
+
+extern sai_object_id_t gVirtualRouterId;
+
+const request_description_t vnet_request_description = {
+    { REQ_T_STRING },
+    {
+        { "src_mac",       REQ_T_MAC_ADDRESS },
+        { "vnet_name",     REQ_T_STRING },
+        { "peer_list",     REQ_T_SET },
+    },
+    { } // no mandatory attributes
+};
+
+enum class VNET_EXEC
```

Comment:
what does VNET_EXEC mean? VNET_TYPE?

Reply:
it means the Vnet execution model. (VRF vs BRIDGE)